src/main/java/frc/robot/commands/climber/ZeroClimber.java

```diff
@@ -0,0 +1,32 @@
+package frc.robot.commands.climber;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.Climber;
+
+public class ZeroClimber extends Command{
+    private Climber climber;
+    public ZeroClimber(Climber climber){
+        this.climber = climber;
+    }
+    
+    @Override
+    public void initialize(){
+        climber.zeroed = false;
+    }
+
+    @Override
+    public void execute(){
+        climber.motionToZero();
+    }
+
+    @Override
+    public void end(boolean interrupted){
+        climber.stopClimber();
+        climber.zeroClimber();
+    }
+    @Override
+    public boolean isFinished(){
+        climber.zeroed = climber.isClimberZeroed();
```

Comment:
This is super bad form. You are telling the state of the climber instance that it's zeroed by asking it if it's zeroed?

If zeroed is a state of the command, make it a field of the command class. If it's a state of the climber, have it owned by the Climber class as private. If you want to set the state of the climber instance inside that class to be the same as isClimberZeroed() than set it as a side effect of isClimberZeroed().